### DISCUSSION TITLE
[Found ~Found] is that a special dataflex indicator?

### DISCUSSION THREAD

**Discussion by SoftAllan** [NilsSve/DfRefactor#68] [General]
In the help file for refactoring functions under Standard - Line-by-Line menu there is a function called ChangeLegacyIndicators. This has some examples where this line occurs:

> [Found ~Found] Begin -> If (Found and Not(Found)) Begin

Will that example ever occur? I mean the same boolean indicator being tested for both a false and a true value at the same time would never case the expression to be true.

**Comment by NilsSve** [comment]
That is a valid code example.

But to answer your question; No, the code within the Begin/End construct will never be executed, but it will be compiled. Look e.g. on line 127 of the DfDafMac.pkg for a sample usage.

The tilde character in the [Found ~Found] expression, means the same as "Not" and is very old DataFlex syntax.
